pkg/server/leases/gc_controller.go

```diff
@@ -70,6 +71,8 @@ func (c *GarbageCollectionController) gc(ctx context.Context) {
 			klog.V(4).Infof("Lease %v was already deleted", lease.Name)
```

Comment:
Could you add HTTP-level counter and latency around this delete call?

Reply:
Done.